costmodel/costmodel.go

```diff
@@ -1142,8 +1215,17 @@ func (cm *CostModel) GetNodeCost(cp costAnalyzerCloud.Provider) (map[string]*cos
 					return nil, err
 				}
 			}
+			if math.IsNaN(nodePrice) {
+				klog.V(1).Infof("[Warning] nodePrice parsed as NaN. Setting to 0.")
+				nodePrice = 0
+			}
 
 			ramPrice := (nodePrice / ramMultiple)
```

Comment:
Should we be dividing here and accepting the nan if this could be 0?

Reply:
So yeah... I thought about this, but it also seemed worthwhile to note each possible problem case rather than skipping some cases knowing they could now be 0 due to our "fixups" Hopefully this will shed some light on the issue.